tests/foreman/cli/test_remoteexecution.py

```diff
@@ -516,6 +517,12 @@ def test_positive_run_receptor_installer(self):
         )
         assert 'project-receptor.satellite_receptor_installer' in result
         assert 'Exit status: 0' in result
+        # check that there is one receptor conf file and it's only readable
+        # by the receptor user and root
+        result = ssh.command('stat /etc/receptor/*/receptor.conf --format "%a:%U"')
+        assert result.stdout[0] == '400:foreman-proxy'
```

Comment:
Now I think about this, if `*` wildcard expands to more than one dir, you will get multiple lines in output, so there would be no way for this assertion to pass. In other words, this assert implicitly contains the next one.

But "Explicit is better than implicit.", so that's OK.